encryption/js-example/Program.js

```diff
@@ -0,0 +1,90 @@
+const crypto = require('crypto');
+const fs = require("fs");
+//Key is 12byte
+const iv = crypto.randomBytes(12);
+//key is 16byte
+var key = "tokopedia1234567";
+var keyBytes = Buffer.from(key)
+var message = '{"hello": "world"}';
+//Public & private key
+//example -----BEGIN PUBLIC KEY-----
+const pubKey = fs.readFileSync("../key/pub.pem", "utf8");
+//example -----BEGIN RSA PRIVATE KEY-----
+const privKey = fs.readFileSync("../key/priv.pem", "utf8");
+
+
+//encryptedPayload
+var encryptedText = encrypt(key,message);
```

Comment:
Im not pro at js, tpi klo kyk gini mendingan masukin ke 1 function aj gk sih kyk di `function main() {} ` trs ntar tinggal panggil aj `main()`